### ISSUE TITLE
NetworkReceiver org.greenstand.android.TreeTracker.network.NetworkReceiver.onReceive(33 crash)

### ISSUE THREAD

**Issue by Surbhi79** [Greenstand/treetracker-android#67]
Fatal Exception: java.lang.RuntimeException: Unable to start receiver org.greenstand.android.TreeTracker.network.NetworkReceiver: android.database.sqlite.SQLiteException: no such table: tree (code 1): , while compiling: SELECT COUNT(*) AS tosync FROM tree WHERE is_synced = 'N'
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:2797)
       at android.app.ActivityThread.access$1800(ActivityThread.java:174)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1401)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:146)
       at android.app.ActivityThread.main(ActivityThread.java:5756)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1291)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1107)
       at dalvik.system.NativeStart.main(NativeStart.java)
[org.greenstand.android.treetracker_issue_5_crash_5AAE75DA011C00015E67916BA106A0FE_DNE_0_v2.txt](https://github.com/Greenstand/treetracker-android/files/1848932/org.greenstand.android.treetracker_issue_5_crash_5AAE75DA011C00015E67916BA106A0FE_DNE_0_v2.txt)

**Comment by Surbhi79** [comment]
Device- Samsung Galaxy A5, Version 1.1.5 (20).

**Comment by beastlyCoder** [comment]
I have a question, looking through the code I think I have identified the problem. 
`			Cursor treeCursor = db.rawQuery("SELECT COUNT(*) AS tosync FROM tree WHERE is_synced = 'N'", null);`

The error message is saying that there is no such table found called tree. Looking through your database code I did not see where you created the table tree. Is this the issue? Or am I just missing where you are instantiating the table tree.
